data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -254,32 +254,6 @@ def _create_tile_data(
             tiles_score_column_titles
         ].copy()
 
-        # Currently, we do not want USVI or Guam on the map, so this will drop all
-        # rows with the FIPS codes (first two digits of the census tract)
-        logger.info(
-            f"Dropping specified FIPS codes from tile data: {constants.DROP_FIPS_CODES}"
-        )
-        tracts_to_drop = []
-        for fips_code in constants.DROP_FIPS_CODES:
-            tracts_to_drop += score_tiles[
-                score_tiles[field_names.GEOID_TRACT_FIELD].str.startswith(
-                    fips_code
-                )
-            ][field_names.GEOID_TRACT_FIELD].to_list()
-        score_tiles = score_tiles[
-            ~score_tiles[field_names.GEOID_TRACT_FIELD].isin(tracts_to_drop)
-        ]
-
-        score_tiles[constants.TILES_SCORE_FLOAT_COLUMNS] = score_tiles[
-            constants.TILES_SCORE_FLOAT_COLUMNS
-        ].apply(
-            func=lambda series: floor_series(
-                series=series,
-                number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
-            ),
-            axis=0,
-        )
```

Comment:
I would leave this code block and just update the comments on 257 - 258 saying something like "these are tracts we drop from the score" or something generic like that.

Reply:
Oh I see, so leave everything but make the constant be an empty array. Sounds good!